### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by eapenzacharias**
- Removed CRA boilerplate contents
- Created React contents to match the given design
- Basic styling added using SCSS
- Imported Calculator class to App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (Davidosky007) [inline src/App.js]: * Kindly Use class-based components here as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.).

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @eapenzacharias  👋, 

Good job so far! 💪 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
* Proper use of git-flow ✔️ 
## There are some issues that you still need to work on to go to the next project but you are almost there!
almost-gif


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (eapenzacharias) [reply src/App.js]: Thank you for pointing it out. I updated `App` to class 
`Calculator` is already mentioned as a class.

https://github.com/eapenzacharias/magic-calculator/blob/e925ca4c6d58e396a79f30b95445900338bb07a0/src/components/Calculator.js#L7-L31

**Review by hiromataba — APPROVED** [review]
# Status: Approved :champagne:  :+1: :confetti_ball: 
Hi @eapenzacharias  :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

Cheers and Happy coding!👏👏👏

## Optional Suggestions

- [ ] [OPTIONAL] Nice commit messages but please use the imperative instead of the past tense

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**